website/docs/sql_dml.md

```diff
@@ -0,0 +1,347 @@
+---
+title: SQL DDL
```

Comment:
Not to reviewer:
for now, sql_dml is just a copy of sql_ddl. 
while I get reviews for rest of the patch, I will update the sql_dml page.